app/routes/_seo+/sitemap[.]xml.ts

```diff
@@ -1,10 +1,10 @@
 import { generateSitemap } from '@nasa-gcn/remix-seo'
-import { routes } from '@remix-run/dev/server-build'
-import { type LoaderFunctionArgs } from '@remix-run/node'
+import { type ServerBuild, type LoaderFunctionArgs } from '@remix-run/node'
 import { getDomainUrl } from '#app/utils/misc.tsx'
 
-export function loader({ request }: LoaderFunctionArgs) {
-	return generateSitemap(request, routes, {
+export async function loader({ request, context }: LoaderFunctionArgs) {
+	const serverBuild = (await context.serverBuild) as ServerBuild
```

Comment:
This was another interesting change because importing `@remix-run/dev/server-build` is not allowed in vite.